### PR TITLE
SD-1400 Create a new highlights/summary package

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: app/scripts/superdesk-packaging/views/highlight.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="subnav">
```
Comment (amagdas) [inline app/scripts/superdesk-packaging/views/highlight.html]: Why is this file needed, doesn't look to be used...

**Comment by amagdas** [comment]
:+1:
